glide.yaml

```diff
@@ -79,3 +79,4 @@ import:
 - package: github.com/jaytaylor/html2text
 - package: gopkg.in/gomail.v2
   version: 2.0.0
+- package: github.com/ssor/bom
```

Comment:
Looks like this project doesn't have a license yet -- I've opened an issue asking if they'd be open to add an MIT or Apache v2 license for the repo: https://github.com/ssor/bom/issues/1